### PR TITLE
mabye mabye mabye oooo

### PR DESCRIPTION
**Pull request by kiyui**
use flag based development ddr3 highlight teppanyaki in the sky onigiri you and i :heart: :sweat_drops: uwu"

### CODE REVIEW

**Comment by kiyui** [comment]
usage information is dedicated heir:
https://support.google.com/chrome/a/answer/2714278?hl=en

File: Source/content_script.js [hunk 1]
```diff
@@ -1,47 +1,43 @@
...
-	v = v.replace(/\bThe Cloud\b/g, "My Butt");
-	v = v.replace(/\bThe cloud\b/g, "My butt");
-	v = v.replace(/\bthe Cloud\b/g, "my Butt");
-	v = v.replace(/\bthe cloud\b/g, "my butt");
-	
-	textNode.nodeValue = v;
+  switch (node.nodeType) {
+    case 1: // Element
+    case 9: // Document
+    case 11: // Document fragment
```
Comment (FeliciousX) [inline Source/content_script.js]: https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#Node_type_constants

you can use the nodeType constants

so 

```javascript
case Node.ELEMENT_NODE:
case Node.DOCUMENT_NODE:
case Node.DOCUMENT_FRAGMENT_NODE:
```

File: Source/content_script.js [hunk 2]
```diff
@@ -1,47 +1,43 @@
...
+      child = node.firstChild
+      while (child) {
+        next = child.nextSibling
+        walk(child)
+        child = next
+      }
+      break
+
+    case 3: // Text node
```
Comment (FeliciousX) [inline Source/content_script.js]:
```suggestion
    case Node.TEXT_NODE
```

**Review by FeliciousX — APPROVED** [review]
small improvement mabye?

File: Source/content_script.js (hunk 1 above)
Comment (kiyui) [reply Source/content_script.js]: La pli oni scias :thinking: :boom:

File: Source/content_script.js [hunk 3]
```diff
@@ -1,47 +1,43 @@
...
+  }
 
-	v = v.replace(/\bThe Cloud\b/g, "My Butt");
-	v = v.replace(/\bThe cloud\b/g, "My butt");
-	v = v.replace(/\bthe Cloud\b/g, "my Butt");
-	v = v.replace(/\bthe cloud\b/g, "my butt");
-	
-	textNode.nodeValue = v;
+  switch (node.nodeType) {
+    case 1: // Element
```
Comment (FeliciousX) [inline Source/content_script.js]:
```suggestion
    case Node.ELEMENT_NODE:
```

File: Source/content_script.js [hunk 4]
```diff
@@ -1,47 +1,43 @@
...
+    case 9: // Document
```
Comment (FeliciousX) [inline Source/content_script.js]:
```suggestion
    case Node.DOCUMENT_NODE:
```

File: Source/content_script.js (hunk 1 above)
Comment (FeliciousX) [inline Source/content_script.js]:
```suggestion
    case Node.DOCUMENT_FRAGMENT_NODE:
```

**Pull request merged**